docker-entrypoint.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/sh
 
-set -e
```

Comment:
We want to fail loud and clear if there's error running this script. This must be restored.

Reply:
The entrypoint changes were not meant to be pushed anyways. Sorry about that.